includes/payments/class-wc-payments-controller.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * REST API Payments Controller
+ *
+ * Handles requests to the /payments endpoint.
+ *
+ * @author   Automattic
+ * @category API
+ * @package  WooCommerce/API
+ */
+
+use Automattic\WooCommerce\Admin\Notes\Note;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * REST API Settings controller class.
+ *
+ * @package WooCommerce/API
+ */
+class WC_Payments_Controller extends WC_REST_Controller {
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc-calypso-bridge/v1';
```

Comment:
I renamed the namespace from `wc/v3` to `wc-calypso-bridge/v1` as we're in wc-calypso-bridge, but we can also keep `wc/v3`.